packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -119,7 +119,10 @@ const SchedulePreviewBody = decorate([
           const log = logs[runId]
           if (
             log.scheduleId === schedule.id &&
-            (lastRunLog === undefined || lastRunLog.start < log.start)
+            (lastRunLog === undefined ||
+              log.status === 'pending' ||
+              (lastRunLog.status !== 'pending' &&
+                (lastRunLog.end || lastRunLog.start) < (log.end || log.start)))
```

Comment:
Simply break if status is pending, I think it's good enough.

Reply:
And keep comparing the `start`s otherwise?

Reply:
I think we can keep `end || start` otherwise.